appsec/src/extension/tags.c

```diff
@@ -1138,6 +1144,46 @@ static PHP_FUNCTION(datadog_appsec_track_custom_event)
     dd_tags_set_sampling_priority();
 }
 
+static PHP_FUNCTION(datadog_appsec_push_params)
```

Comment:
this has nothing to do with tags, so shouldn't be in this file

Reply:
I have moved this to `ddappsec.c`